config.ru

```diff
@@ -0,0 +1,14 @@
+require 'puppet_webhook'
+require 'logger'
+
+LOGGER = Logger.new('/var/log/puppet_webhook').freeze
+COMMAND_PREFIX = 'umask 0022;'.freeze
```

Comment:
this constant isn't used

Reply:
Are you referring to COMMAND_PREFIX?

It's used in the two of the helper files:
The `deploy_module` and `deploy` methods in `helpers/deployments.rb` and the `generate_types` method in `helpers/tasks.rb` use that constant.

Now...... whether that _should_ be a constant is another another discussion that should happen.